src/main/java/nl/nn/testtool/util/Export.java

```diff
@@ -105,7 +115,13 @@ private static ExportResult export(Storage storage, Report report,
 				Set duplicateCheck = new HashSet();
 				Iterator iterator = storageIds.iterator();
 				while (iterator.hasNext()) {
+					try {
 					report = storage.getReport((Integer)iterator.next());
+						if (report == null)
+							continue;
+					} catch (Exception e) {
```

Comment:
Should we just ignore any error? ExportResult has setErrorMessage(), maybe we should use that method?

Reply:
I have added the setErrorMessage to the exception